lemur/plugins/lemur_aws/plugin.py

```diff
@@ -267,7 +269,11 @@ def update_endpoint(self, endpoint, certificate):
 
         # relies on the fact that region is included in DNS name
         region = get_region_from_dns(endpoint.dnsname)
-        arn = iam.create_arn_from_cert(account_number, region, certificate.name)
+        if endpoint.registery_typ == 'iam':
+            arn = iam.create_arn_from_cert(account_number, region, certificate.name, endpoint.certificate_path)
```

Comment:
a more accurate `arn`, should help with catching any corner cases, and help with a better understanding about all discovered endpoints.

Reply:
this also captures the registry_type` NoneType case.